_data/simple-icons.json

```diff
@@ -788,6 +788,11 @@
             "hex": "49B48A",
             "source": "https://www.arlo.com/"
         },
+        {
+            "title": "Artifact Hub",
+            "hex": "417598",
+            "source": "https://artifacthub.io/static/media/logo/artifacthub-brand-white.svg"
```

Comment:
I found the source in their Github repo. Can you change it for:
https://github.com/artifacthub/hub/blob/537e4862d0aa0a689eaa66e06354c1fb84eaf134/web/public/static/media/logo/artifacthub-brand-white.svg